benchmarks/mpas_ocean.py

```diff
@@ -81,8 +81,6 @@ class GeoDataFrame(DatasetBenchmark):
     def time_to_geodataframe(self, resolution, exclude_antimeridian):
         self.uxds[data_var].to_geodataframe(exclude_antimeridian=exclude_antimeridian)
 
-    def peakmem_to_geodataframe(self, resolution, exclude_antimeridian):
-        gdf = self.uxds[data_var].to_geodataframe(exclude_antimeridian=exclude_antimeridian)
```

Comment:
> Note
> 
> The peak memory benchmark also counts memory usage during the setup routine, which may confound the benchmark results. One way to avoid this is to use setup_cache instead.

From the ASV documentation. I can include it back, however I chose to remove it since we aren't correctly implementing it.

Reply:
Thanks for clarifying! No need to add it back